example/declarative.dart

```diff
@@ -1,29 +1,31 @@
 // Copyright 2013 Google. All rights reserved. Use of this source code is
 // governed by a BSD-style license that can be found in the LICENSE file.
 
-library grinder_example_1;
+library grinder.example.declarative;
 
 import 'package:grinder/grinder.dart';
 
-void main([List<String> args]) {
-  task('init', init);
-  task('compile', compile, ['init']);
-  task('deploy', deploy, ['compile']);
-  task('default', null, ['deploy']);
-
+void main(List<String> args) {
```

Comment:
Re: `addAnnotatedTasks()` - we may want to prefer the annotation mechanism as the best way to run grinder. The programmatic way would exists for people who needed more control over the machinery, but the declarative way would be the 90% case.

So, is there a good way to discover tasks w/o an explicit call to `addAnnotatedTasks()`?
```
main() => startGrinder();

@task
doFoo() {
...
}

@task(depends: 'foo')
doBar() {
...
}
```